studio/schemas/builders/pageBuilder.ts

```diff
@@ -17,13 +17,13 @@ export const pageBuilderID = "pageBuilder";
 const pageBuilder = defineType({
   name: pageBuilderID,
   type: "document",
-  title: "Pages",
+  title: "Dynamic pages",
   fields: [
     defineField({
       name: "page",
-      title: "Page name",
+      title: "Dynamic page name",
```

Comment:
I think we can keep this to "Page name" as it's a page within dynamic pages, so it's pretty clear its a dynamic page